src/cycax/cycad/cycad_side.py

```diff
@@ -23,8 +22,7 @@ def hole(
             pos : this is a tupple that contains the (x, y) coordinates of the object.
             diameter: The diameter of the hole.
             depth: How deep to drill the hole, if not specified will drill the hole all the way through.
-            inner: This is specified that the hole is internal and will not be coppied onto other surface. Defaults to True.
-            external_only: This is specified that the hole will only be tranferred onto other surfaces and will not be drilled into main object. Defaults to False.
+            external_subtract: This is specified that the hole will only be tranferred onto other surfaces and will not be drilled into main object. Defaults to False. When set to True the hole will not be drilled into the main object.
```

Comment:
Please have a look at the generated docs (`make docs`). I expect that mkdocs would add the default for you, this no need to mention.